src/client/entity-editor/button-bar/button-bar.js

```diff
@@ -56,43 +55,30 @@ import {connect} from 'react-redux';
  */
 function ButtonBar({
 	aliasesInvalid,
-	disambiguationVisible,
 	identifiersInvalid,
 	numAliases,
 	numIdentifiers,
 	onAliasButtonClick,
-	onDisambiguationButtonClick,
```

Comment:
There is a leftover `onDisambiguationButtonClick` in the JSDOC documentation comments on lines 50-51